packages/sharec-core/lib/steps/readConfigs.js

```diff
@@ -3,6 +3,7 @@ const { readFile } = require('sharec-utils').std
 const { join } = require('sharec-utils').path
 const { find } = require('sharec-utils').fs
 const { InternalError, errorCauses } = require('../errors')
+const binaryExtensions = require('binary-extensions')
```

Comment:
Binary file can be without any extension

Reply:
I will replace it with the library [istextorbinary](https://www.npmjs.com/package/istextorbinary)